croptalk/retriever.py

```diff
@@ -75,7 +75,7 @@ class RetrieverInput(BaseModel):
     county: Optional[str] = Field(description="County name. Example: Ventura")
 
 
-def format_docs(docs: Sequence[Document]) -> str:
+def format_docs(docs: Sequence[Document]) -> List[str]:
```

Comment:
Is this function still used??
It seems to be using the chromadb retriever, no?

argh... we need a bit of a cleanup I think!

Reply:
indeed, I was just looking at the code and made a few changes